src/Core/Domain/Cart/CartStatus.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\Cart;
+
+/**
+ * Cart status
+ */
+class CartStatus
+{
+    /**
+     * Cart ordered
+     */
+    public const ORDERED = 'ordered';
+
+    /**
+     * Cart not ordered
+     */
+    public const NOT_ORDERED = 'not_ordered';
+
+    /**
+     * Cart not ordered but for long time
+     */
+    public const ABANDONED_CART = 'abandoned_cart';
+
+    /**
+     * Time in seconds representing time before not ordered carts are considered abandoned
+     * (For now: 24h)
+     */
+    public const ABANDONED_CART_EXPIRATION_TIME = 86400;
```

Comment:
Oh, also as this class is only used to store constants (and there is no Create/Edit cart commands where we would use it as VO), it could have a private constructor, so that it is clear that its not intended to be instantiated. Like in this similar purpose class https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Domain/Product/ProductSettings.php#L40